src/tests/teamcity-event-listener.tests.csproj

```diff
@@ -28,6 +29,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <LangVersion>3</LangVersion>
```

Comment:
If the extension needs newer features of C#, then we should simply document that somewhere. It only becomes a problem if somebody is trying to build it on an older version of the compiler, so it would be good for them to get an intelligent error message. Seems like it only affects nunit developers, not users.